src/guacamole-docker.md

```diff
@@ -782,6 +786,42 @@ SAML authentication extension.
   that may be helpful in debugging problems with SAML authentication. This
   is optional and defaults to false - debugging will not be enabled.
 
+(guacamole-docker-history-recording-storage)=
+
+### History Recording Storage Extension
+
+The extension that enables viewing historical recordings from within the
+Guacamole Client interface can be enabled by settings the search path
+variable, as noted below, to a location where the extension will look
+to find available recordings.
+
+When setting this up in a container environment, you'll likely need
+to use volumes to make the same directory available to both the
+guacd container and the guacamole (client) container. In addition
+to setting up the volume to share data between the two, you'll also
+need to configure permissions on the volume such that the users
+running each of the containers have access. The guacd container,
+which will need write access to this shared location, runs with the
+effective UID and GID of 1000. The guacamole-client container, which
```

Comment:
I recommend referring to these by their Docker Hub image names (`guacamole/guacd` and `guacamole/guacamole` respectively).

Reply:
Ah, yes...fixed via rebase.